### ISSUE TITLE
Offline comm configuration

### ISSUE THREAD

**Issue by KingRaptor** [ZeroK-RTS/Chobby#364]
ZK should have offline comm configuration within Chobby. The current website configuration is prone to bugs, difficult to maintain/update and can't be used offline.
(see ZeroK-RTS/Zero-K-Infrastructure/issues/1209, ZeroK-RTS/Zero-K-Infrastructure/issues/1749)

Requirements:

- [ ] Comm config GUI in Chobby menu that saves to file
- [ ] Give ZK startup comm selector the ability to read aforementioned file and display comms
- [ ] Let server transmit unlocks list to client (for kudos decorations), possibly via Chobby wrapper

As a QoL bonus:

- [ ] One-time transmission of existing website configs to local user

Campaign has a comm configurator that could be adapted for this general use.

Online configuration could be kept as a fallback for the time being.

**Comment by GoogleFrog** [comment]
I don't think this should have the steam milestone, given its complexity. This feature has no effect on campaign and has little-to-no effect on other first impressions. Later on people learn that to give their commanders names and decorations they need to use the site.

**Comment by Licho1** [comment]
What is the aim we are trying to solve again here? 
What do you mean exactly?
Comm configuration exists on the client .. explain please

**Comment by deee167** [comment]
I would like this feature implemented so I can configure my comm in game and choose to use any of my saved presets all saved locally and used locally without needing an online connection.

Thanks
